tests/base/model_train_steps.py

```diff
@@ -21,12 +21,21 @@ def training_step(self, batch, batch_idx, optimizer_idx=None):
 
         # calculate loss
         loss_val = self.loss(y, y_hat)
+        loss_scalar = loss_val.item()
 
         # alternate possible outputs to test
+        if batch_idx % 2 == 0:
+            output = OrderedDict({
+                'loss': loss_val,
```

Comment:
let's write this if else here also in the style like I show in the other comment below